test/helpers.js

```diff
@@ -153,19 +153,20 @@ Runner.prototype.npmVersion = function(version){
 	if(arguments.length === 0) {
 		return this._version; 
 	}
-
-	this.algorithm = version >= 3 ? "flat": undefined;
 	this._version = version;
-	this._addVersion();
```

Comment:
Why was this removed?  Now `_addVersion` is only called when rootPackage is called, so if you did something like:

```js
helpers.clone()
  .rootPackage({
     ...
  })
  .npmVersion(2)
```

_addVersion wouldn't run.